src/util/globals.js

```diff
@@ -201,9 +201,9 @@ export const isValueValid = (value) => {
   // booleans, valid strings & numbers are valid.
   return true;
 };
-export const strainSymbol = Symbol('strain');
-export const genotypeSymbol = Symbol('genotype');
-export const measurementIdSymbol = Symbol('measurementId');
+export const strainSymbol = 'strain';
```

Comment:
I presume this change is to make types easier?

I'd prefer to keep symbols - they are perfect for this use case. One downside to them is that they are skipped with things like `Object.keys()`, but I'd hope TS would be kind enough to flag this up for us?

Reply:
Yeah mainly for ease of typing. I can look into working with them as-is.

Reply:
Addressed with with 3c3ef2dcabf6e5af70f9460eeefbeaf3dbc36363